sql/sql_acl.cc

```diff
@@ -2101,9 +2101,10 @@ acl_init_utility_user(my_bool check_no_resolve)
       if (*cur_pos == ',' || *cur_pos == '\0')
       {
         char *dbname= my_strndup(cur_db, cur_pos-cur_db, MYF(MY_FAE));
-        (void) push_dynamic(&acl_utility_user_schema_access, (uchar*) &dbname);
+	if (strlen(dbname) > 0)
```

Comment:
Sorry, @dutow 
this is a memory leak if `strlen(dbname) == 0`
May be something like in 5.7
```
if (cur_pos - cur_db > 0)
{
  char *dbname= my_strndup(cur_db, cur_pos-cur_db, MYF(MY_FAE));
  (void) push_dynamic(&acl_utility_user_schema_access, (uchar*) &dbname);
}
```

Reply:
Fixed.